### ISSUE TITLE
[Unblock Legitimate Site]

### ISSUE THREAD

**Issue by poocoinapp** [MetaMask/eth-phishing-detect#13721]
**Domain:**
https://poocoin.app

**Details:** *Some details about the domain (optional)*

The site is a blockchain reading tool that reads price charts for pancakeswap tokens and shows a list of tokens in your connected wallet. It has recently started getting flagged as malicious by the metamask plugin.

- [x ] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
this is not flagged or blocked. I am unable to reproduce this. Can you please clarify?

**Comment by poocoinapp** [comment]
The warning appeared once yesterday but no longer appeared after I dismissed it. Some people in the sites telegram channel reported the same issue yesterday. I cant reproduce it anymore even on a different computer. It might have been only blocked temporarily if nobody can reproduce it anymore.

**Comment by AlexHerman1** [comment]
okay, if it continues to pop up please open a new issue and we will look into it.
